src/vic3_world/countries/vic3_country.h

```diff
@@ -30,6 +30,15 @@ struct CountryOptions
    std::set<int> puppets;
 };
 
+enum RankCategory
+{
+   RANK_GREAT_POWER,
+   RANK_MAJOR_POWER,
+   RANK_MINOR_POWER,          // called "unrecognized regional power" if unrecognized
+   RANK_INSIGNIFICANT_POWER,  // called "unrecognized power" if unrecognized
+   RANK_DECENTRALIZED_POWER
```

Comment:
Use naming more like the Google style guide recommends: https://google.github.io/styleguide/cppguide.html#Enumerator_Names

You can choose to have the k prefix or not.